source/How-To-Guides/Use-The-Dashboard.rst

```diff
@@ -1,7 +1,96 @@
-Using use the Space ROS Dashboard
-=================================
+#############################
+Using the Space ROS Dashboard
+#############################
 
-TODO:
+Static analysis is a critical step in the formal verification required for safety-critical systems.
+The Space ROS Dashboard is a tool for aggregating and tracking issues discovered through static analysis.
+These issues are represented in `SARIF format <https://sarifweb.azurewebsites.net/>`_, which is a format for representing issues found by static analysis tools.
 
-How to use the Space ROS Dashboard, including installation of the VSCode plug-in.
+The main features of the Space ROS Dashboard are as follows:
 
+- Track the dispositioning issues, including annotating them with links to bug reports in your dispositioning service of choice
```

Comment:
"Track the dispositioning issues" -> "Track issues reported by static analysis tooling"